app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/ProfilesButton.tsx

```diff
@@ -9,38 +9,39 @@ import i18n from "../../utils/i18n";
 export interface ProfilesButtonProps {
     profile: Profile;
     onChange(newProfile: Profile): void;
+    expanded: boolean;
 }
 
-export const ProfilesButton: React.FC<ProfilesButtonProps> = React.memo(props => {
-    const { profile, onChange } = props;
-
-    const dropdownItems: DropdownProps<ProfileId>["items"] = React.useMemo(() => {
-        return _.values(profiles).map(profile => {
-            return { text: profile.name, id: profile.id };
-        });
-    }, []);
-
-    const setProfile = React.useCallback(
-        (profileId: ProfileId) => {
-            sendAnalytics("set_profile", {
-                on: "viewer",
-                label: profileId,
+export const ProfilesButton: React.FC<ProfilesButtonProps> = React.memo(
+    ({ profile, onChange, expanded }) => {
```

Comment:
[subjective] Not a big fan of in-line props because of readability and also what happened here, it added one level of indentation.

Reply:
Yeah it's true, my bad